platforms/cpuinfo_test.go

```diff
@@ -0,0 +1,25 @@
+package platforms
+
+import (
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func TestCPUVariant(t *testing.T) {
+	if !isArmArch(runtime.GOARCH) || !isLinuxOS(runtime.GOOS) {
+		return
+	}
+
+	variants := []string{"v8", "v7", "v6", "v5", "v4", "v3"}
+
+	p := getCPUVariant()
+	for _, variant := range variants {
+		if reflect.DeepEqual(p, variant) {
```

Comment:
No need for deepequal here. This is a string.

Reply:
OK